src/views/authenticated/List.tsx

```diff
@@ -25,9 +27,7 @@ export function List({ data: unfilteredListItems, listPath }: Props) {
 	if (unfilteredListItems.length === 0) {
 		return (
 			<>
-				<p>
-					Hello from the <code>/list</code> page!
-				</p>
+				<p>{listName}</p>
```

Comment:
nitpick (non-blocking): we could change this to some type of header tag. it maybe would mean adjusting the others on the page since <h1> is used in the header but it may make it a more distinct part of the page.

Reply:
Yes I agree! I do not think the listName should be in a paragraph element, that's very disrespectful to it!!!

Reply:
I didn't realize it would put that text in as a header 😂